pypi_resource/pipio.py

```diff
@@ -45,15 +45,17 @@ def run(self, options, args):
         options.retries = RETRIES
         options.ignore_installed = True
         options.editables = []
+        options.platform = None
+        options.abi = None
 
         with self._build_session(options) as session:
             finder = self._build_package_finder(
                 options=options,
                 session=session,
                 target_python = TargetPython(
```

Comment:
TargetPython (`from pip._internal.models.target_python import TargetPython`)  changed parameter names in newer pip versions.